src/gui/components/ChannelView.tsx

```diff
@@ -104,10 +109,16 @@ export default memo(function ChannelView({
         if (!client.setPeers) {
             return;
         }
-        client.setPeers(channelId, potentialPeers).catch((err) => {
-            console.error('acceptPeers', err);
+
+        const selectedPeers = [peerId, ...peers.map((p) => p.peerId)].slice(
+            0,
+            MAX_PLAYERS,
+        );
+
+        client.setPeers(channelId, selectedPeers).catch((err) => {
+            console.error('acceptPeers error:', err);
         });
-    }, [client, channelId, potentialPeers]);
+    }, [client, channelId, peerId, peers]);
```

Comment:
non-blocking: what you have done looks correct ... but I wonder if we should make potentialPeers reflect what the selected peers will be? .... that way it's not a surprize who gets in?

Reply:
This commit adds grey names for people who won't make it in and is consistent on all clients: https://github.com/playmint/playerchain-demo/pull/117/commits/831011b75b8b6cc397e44a692d66651fe1bbd843

Just struggling to make any good progress with the PacketLace fix.

Reply:
if it's too much of a pain for the packetlace to be "roughly right", then let's just make sure it never goes above 4... the prioirty is that it doesn't look broken by attempting to show more than fit

Reply:
Works via latest commit! https://github.com/playmint/playerchain-demo/pull/117/commits/20805e0414dee8db26441d83dd474beba14db620

Now: 
- same behavior before the game starts
- after go is typed, it will then only show the peers of the players still in game (up to `MAX_PLAYERS`)